tests/foreman/api/test_contentview.py

```diff
@@ -1180,6 +1181,35 @@ def test_positive_promote_out_of_sequence(self):
         self.assertEqual(len(content_view.version[0].read().environment), 1)
         self.assertEqual(len(content_view.version[-1].read().environment), 0)
 
+    @tier2
+    def test_positive_publish_multiple_repos(self):
+        """Attempt to publish a content view with multiple RPM repos.
+
+        :id: 5557a33b-7a6f-45f5-9fe4-23a704ed9e21
+
+        :expectedresults: Content view publish should not raise an exception.
+
+        :CaseLevel: Integration
+
+        :CaseComponent: Pulp
+
+        :CaseImportance: Medium
+
+        :BZ: 1651930
+        """
+        org = entities.Organization().create()
+        product = entities.Product(organization=org).create()
+        content_view = entities.ContentView(organization=org).create()
+        for _ in range(10):
+            repo = entities.Repository(product=product).create()
+            repo.sync()
+            content_view.repository.append(repo)
+            content_view = content_view.update(['repository'])
+        content_view = content_view.read()
+        self.assertEqual(len(content_view.repository), 10)
+        with self.assertNotRaises(TaskFailedError):
+            content_view.publish()
+
```

Comment:
Could you please add assertion after publish as well ..
i.e.
```
self.assertEqual(len(content_view.read().version), 1)
```

Reply:
@vijay8451, thanks for your review. Updated per comments.

Reply:
thanks @kersommoura